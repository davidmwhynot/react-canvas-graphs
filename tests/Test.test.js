import React from 'react';
import { Test } from '../build/index.module';
import renderer from 'react-test-renderer';

test('Test', () => {
	const component = renderer.create(
		<div>
			<Test>Test</Test>
		</div>
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
