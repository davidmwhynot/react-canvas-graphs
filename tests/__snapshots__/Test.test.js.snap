// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test 1`] = `
<div>
  <div
    style={
      Object {
        "padding": "3px 7px",
      }
    }
  >
    <span>
      Test
    </span>
  </div>
</div>
`;
